api/custom_auth/views.py

```diff
@@ -51,3 +54,22 @@ def delete_token(request):
         sender=request.user.__class__, request=request, user=request.user
     )
     return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class CustomUserDeleteView(UserViewSet):
+    def destroy(self, request, *args, **kwargs):
+        if "delete_orphan_organizations" in request.query_params:
+            delete_orphan_organizations = request.query_params[
+                "delete_orphan_organizations"
+            ]
+            if delete_orphan_organizations.lower() == "true":
+                FFAdminUser.delete_orphan_organisations(request.user)
```

Comment:
For some reason, we're calling this as if it's a static method but it's not... this needs revisiting.

Reply:
Corrected